markdown/blog/2024-marketing-report.md

```diff
@@ -0,0 +1,187 @@
+By aligning our goals with AsyncAPI’s mission, the MWG seeks to empower both seasoned and new members to engage deeply with the initiative. 
+
+Below is a summary of what we’ve accomplished this year and our priorities moving forward.
+
+## Accomplishments and Milestones
+We’ve focused on laying the groundwork for long-term growth and engagement for the last six months. Our marketing initiatives have been designed to elevate AsyncAPI’s visibility, simplify our communication strategy, and ensure we’re prepared to grow alongside our community. 
+
+Here’s a summary of what we’ve done so far:
+- Launched targeted campaigns for our programs and events.
+- Held successful online events that engaged and informed over 350 community members and prospective contributors
+- Implemented regular content updates on our platforms to keep followers informed and engaged
+- Initiated the Student Ambassador Programs that will bring more attention to AsyncAPI’s mission and tools, further building our presence and strengthening community ties
+
+## Implementing Buffer for Social Media Management
+
+To improve the efficiency and security of our social media management, we’ve introduced a paid version of Buffer for posting content across our platforms. This change enables us to streamline content scheduling, ensuring consistent, professional updates without compromising security. Importantly, using Buffer restricts access to the main social media accounts, adding a layer of protection and allowing us to manage permissions efficiently.
+
+Here are some of the benefits of using the Buffer subscription
+- **Centralized Dashboard:** Manage all social media platforms from a single interface, making tracking and organizing content easier. 
+- **Improved Security:** No need for multiple people to log into the actual social media accounts. This minimizes the risks of unauthorized changes or breaches.
+- **Better Analytics and Insights:** Access in-depth analytics to understand which posts perform and engagement metrics best and adjust strategies accordingly. 
+- **Content Approval Workflow:** Allow team members to draft posts for review, ensuring high-quality content that aligns with AsyncAPI’s voice.
+
+
+Please note that I currently do not have direct access to AsyncAPI’s social media accounts. Buffer provides an effective solution for posting content, monitoring performance, and maintaining the high standards of AsyncAPI's public presence without broad access.
+
+As part of the benefit of using Buffer, we have access to our social media analytics. Below are the analytics for LinkedIn and X (formally Twitter) from May 20th to December 10th, 2024.
+
+
+![AsyncAPI LinkedIn engagement metrics showing follower growth and post metrics from Buffer analytics](/img/posts/marketing-images/linkedIn-metrics-insights.webp)
```

Comment:
now that I look at the diagrams again

maybe better would be diagram that shows stuff starting from January, and then in diagram you can mark the part when you started handling socials, so the differentce is shows in diagrams as well, not numbers only